public/app/plugins/datasource/grafana-azure-monitor-datasource/resourcePicker/supportedResources.ts

```diff
@@ -0,0 +1,183 @@
+// TODO: should this list be different for logs vs metrics?
```

Comment:
Good shout.

We'll need to see if we can get this list from Azure itself in the near future. I would rather not have to maintain this.